sdk/Lusid.Sdk.Tests/tutorials/Ibor/Orders.cs

```diff
@@ -321,34 +320,35 @@ public void Upsert_And_Retrieve_Simple_Orders()
 
             var quantityFilter = _ordersApi.ListOrders(asAt:
                 t,
-                filter: $"Quantity gt 100 and Scope eq '{testScope}' and Id in '{order1Filter}', '{order2Filter}', '{order3Filter}'");
+                filter: $"Quantity gt 100 and Id.Scope eq '{testScope}' and Id.Code in ('{order1Filter}', '{order2Filter}', '{order3Filter}')");
```

Comment:
`order1Filter` etc  look like they're in the from "{Scope}/{Code}"; if we only want to filter by Id.Code, the filters will need to be changed to "{Code}"

Reply:
Amended to meet above inputs:
> var order1Filter = $"{testScope}/{order1}"; --> var order1Filter = $"{order1}";